### ISSUE TITLE
[Feature Request] Toggle for reverse order of auto-playlists

### ISSUE THREAD

**Issue by KurtTrowbridge** [radiant-player/radiant-player-mac#479]
As one of many people migrating from Rdio to Google Play Music, I'm accustomed to a list of favorites/thumbs-up tracks that are in reverse-chronological order, with the newest likes at the top of the list.  In GPM, the first songs with a thumbs-up are listed first in the *Thumbs up* auto-playlist, meaning I have to scroll down to get to the newest ones. Would it be possible to add either a column (like the track numbers on playlists) or a menu option that would let me reverse the order of *Thumbs up* and other auto-playlists?

Thanks for your consideration!

**Comment by chrismou** [comment]
Hey @KurtTrowbridge -- I think it already shows newest at the top?  At least it does on mine

**Comment by KurtTrowbridge** [comment]
Ah, I think you're right, @chrismou. I had tried giving a thumbs-up to multiple tracks right after each other, and they weren't showing in the right order (I just did five in fairly close succession and they show in the order 5/3/1/4/2, where 5 is the last one I liked), but it looks like they're correctly in reverse-chronological order after refreshing Radiant Player. Thanks for the extremely fast response!
